src/app/components/modal/index.html

```diff
@@ -118,6 +117,11 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Modal instance events">
+    <sky-demo-page-property
+      propertyName="beforeClose"
+    >
+      An event that the modal instance emits when it is about to close. It emits a <stache-code>SkyModalBeforeCloseHandler</stache-code> object with a <stache-code>closeModal</stache-code> method that completes the close of the modal. The modal will not close unless the <stache-code>closeModal</stache-code> method is called if there is a subscription to this event.
```

Comment:
In the second sentence, would it change the meaning if we change "that completes the close of the modal" to "that closes the modal"?

In the last sentence, I'd suggest moving the if clause to the start of the sentence and tweaking to use active voice: "If a subscription exists for this event, the modal does not close until that description calls the `closeModal` method." Does that work?

Reply:
*subscriber calls the

But yes these changes look good 🙂